packages/react-material-ui/src/components/Text/Text.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import Typography, { TypographyProps } from '@mui/material/Typography';
 
+/**
+ * The Text component is a wrapper around the Material-UI Typography component
```

Comment:
Please put a link to the MUI pieces like we do on this one..

https://github.com/conceptadev/rockets-react/blob/main/packages/react-material-ui/src/components/AppBar/AppBarMain.tsx